### PR TITLE
DOC-10496 - Document Platform Root CA Usage

### PR DESCRIPTION
**Pull request by maria-robobug**

### CODE REVIEW

File: modules/howtos/pages/managing-connections.adoc [hunk 1]
```diff
@@ -111,11 +112,34 @@ NOTE: Any TLS certificates must be set up at the point where the connections are
...
+--
+As of SDK 3.4, if you connect to a Couchbase Server cluster with a root certificate issued by a trusted CA (Certificate Authority), you no longer need to configure the `trust_certificate` path in your connection string.
+
+The cluster's root certificate just needs to be issued by a CA whose certificate is in your system trust store.
+This includes well known CAs (e.g., GoDaddy, Verisign, etc...), plus any other CA certificates that you wish to add.
+
+IMPORTANT: The SDK uses an https://github.com/ruby/openssl[OpenSSL wrapper] for TLS functionality, which loads your system's certificate store by default.
+If your system does not have a default root CA store, it will use the https://ruby.github.io/openssl/OpenSSL/X509/Store.html[Mozilla CA] certificate store.
+
+You can still provide a certificate explicitly if necessary:
```
Comment (maria-robobug) [inline modules/howtos/pages/managing-connections.adoc]: @avsej or @DemetrisChr, could you have a look at this when you have a minute?

Just want to make sure this is correct!

Comment (DemetrisChr) [reply modules/howtos/pages/managing-connections.adoc]: I think I would replace "well known CAs" with "publicly trusted CAs" (not sure about this though).

We are not using the Ruby OpenSSL wrapper, the functionality is implemented in the C++ SDK core, where we retrieve the platform's root CA store in a similar way to the Ruby OpenSSL wrapper. @avsej might be able to give more details on this.

**Comment by maria-robobug** [comment]
👍 thanks for looking at this @DemetrisChr, I've made some slight changes based on your review.
Will wait to see if @avsej is happy with this as well.

**Review by avsej — APPROVED** [review]

**Comment by avsej** [comment]
Looks good to me, thanks @DemetrisChr  on that note.

**Pull request merged**